### PR TITLE
Tests for AffineScore

### PR DESCRIPTION
**Pull request by MatheMax**

### CODE REVIEW

File: R/AffineScoreFunction.R [hunk 1]
```diff
@@ -29,7 +29,7 @@ AffineScore <- function(scores, coefs, intercept) {
         stop("intercept must have length 1")
     if (any(!is.finite(c(coefs, intercept))))
         stop("scores and intercept must be finite")
-    new("AffineScore", scores = scores, coefs = coefs, intercept = intercept)
+    new("AffineScore", scores = c(scores), coefs = coefs, intercept = intercept)
```
Comment (kkmann) [inline R/AffineScoreFunction.R]: what's happening here? why do we need the c()? where did it break down?

**Comment by kkmann** [comment]
there also still seem to be a few todo notes and missing documentation - would probably be better to do it in one wash. If it's too complicated, just comment here and open an issue so that we can keep track of things. Crucial stuff goes to the 0.1 milestone, everything else to 0.2 or none ;)

**Comment by kkmann** [comment]
I tentatively put March 7th as target date for 0.1, pretty good progress so far https://github.com/kkmann/otsd/milestones

File: R/AffineScoreFunction.R (hunk 1 above)
Comment (MatheMax) [reply R/AffineScoreFunction.R]: When scores just has one element it needs to be converted to a vector.
Returned an error, e.g., here:
https://github.com/kkmann/otsd/blob/69c15f2fafaaa7d0f4210a2ee458eade9ececbc6/tests/testthat/test_AffineScoreFunction.R#L39

Comment (kkmann) [reply R/AffineScoreFunction.R]: Cool! And it now works and both cases are tested?

Comment (MatheMax) [reply R/AffineScoreFunction.R]: Yes as we are adding scores in the minimization 
But I can write a specific test for this!

Comment (kkmann) [reply R/AffineScoreFunction.R]: please! makes it easier to identify if we break it again ;)

Comment (kkmann) [reply R/AffineScoreFunction.R]: whenever you fix something plz add a test case making sure it's not going to happen again.

Comment (MatheMax) [reply R/AffineScoreFunction.R]: Alright! I also add tests to achieve 100% coverage

File: tests/testthat/test_AffineScoreFunction.R [hunk 2]
```diff
@@ -76,20 +85,36 @@ test_that("Arithmetic for conditional scores works", {
...
+    ) # multiplication by 1 does not change anything
 
     expect_equal(
         incpt * afcosc,
         afcosc
-    ) # multipilcation is commutative
+    ) # multiplication is commutative
+
+    expect_equal(
+        evaluate((-1) * afcp + afcosc, design, z1),
```
Comment (kkmann) [inline tests/testthat/test_AffineScoreFunction.R]: do we need the brackets here? Does `-1 afcp` work? We should create an issue for getting -score to work at some point (non-priority)...

Comment (kkmann) [reply tests/testthat/test_AffineScoreFunction.R]: #64

**Pull request merged**